README.md

```diff
@@ -5,24 +5,27 @@ Alacritty
 [![Appveyor build Status](https://ci.appveyor.com/api/projects/status/github/jwilm/alacritty?svg=true)](https://ci.appveyor.com/project/jwilm/alacritty)
 
 Alacritty is the fastest terminal emulator in existence. Using the GPU for
-rendering enables optimizations that simply aren't possible in other emulators.
-Alacritty currently supports macOS, Linux, and Windows.
+rendering enables optimizations that simply aren't possible without it.
+Alacritty currently supports macOS, Linux, BSD, and Windows.
 
 <p align="center">
-  <img width="600" alt="Alacritty running vim inside tmux" src="https://cloud.githubusercontent.com/assets/4285147/21585004/2ebd0288-d06c-11e6-95d3-4a2889dbbd6f.png">
+  <img width="600"
+       alt="Alacritty running vim inside tmux"
+       src="https://cloud.githubusercontent.com/assets/4285147/21585004/2ebd0288-d06c-11e6-95d3-4a2889dbbd6f.png">
 </p>
 
 ## About
 
-Alacritty is focused on simplicity and performance. The performance goal means
-it should be faster than any other terminal emulator available. The simplicity
-goal means that it doesn't have features such as tabs or splits (which can be
-better provided by a window manager or [terminal multiplexer][tmux]) nor
-niceties like a GUI config editor.
+Alacritty is a terminal emulator with a strong focus on simplicity and
+performance. Bloated features are avoided, and the features we do include are
+carefully considered. This means you won't find things like tabs or splits
+(which are best left to a window manager or [terminal multiplexer][tmux]) nor
+niceties like a GUI config editor. We also strive to keep alacritty faster than
+any other terminal available. Performance matters!
```

Comment:
I don't like the way this reads since it kinda feels like it's mostly `Alacritty is not [...]`, rather than `Alacritty is [...]`. While it's important to state what is not the focus of Alacritty, I feel like we should make it clear what the advantages of Alacritty are first.

I'm also not sure if I'd mention specific features like tabs/splits/GUI config editor in the about section, Alacritty is not *about* not having those things. Not having those things is just a result of what Alacritty is about.

I'm a big fan of the first sentence, that already gives a pretty good idea of what Alacritty is. But I think we can tweak the rest a bit more to showcase what Alacritty is for. Then maybe we could put the tabs/splits/config editor in the FAQ section?

Reply:
I agree about moving the bit about tabs and splits into the FAQ, I was frankly just being conservative with my edits here. I'm tempted to let this about section shrink a little as a result. See updated commit incoming for another draft.

Reply:
Actually decided to try adding a sentence about defaults and configuration. Let me know.